cmd/grpc.go

```diff
@@ -137,6 +139,12 @@ func (s *server) RunFunction(ctx context.Context, in *pb.RunFunctionRequest) (*p
 	if err != nil {
 		return nil, status.Errorf(codes.Aborted, "cannot enable devMode: %s", err)
 	}
+	if ExternalConnectionEnabled {
```

Comment:
Instead of conditionally adding this, just add it to the list like all other functions.

Then the first thing you do in the function is checking if the loadbalancer config is enabled or not and either just return or run the function.